RELEASE.md

```diff
@@ -40,11 +42,17 @@ Close the release on JIRA, adding the current date (you may need to ask the SQL
 Ensure you have the `master` branch checked out, and that you have pulled the latest commit from `mongodb/mongo-jdbc-driver`.
 
 #### Create the tag and push
+
+To determine which version of [`libmongosqltranslate`](https://github.com/10gen/mongosql-rs) to build with the release, the version
```

Comment:
nit: The wording here sounds like I need to determine something where it's really means to specify the version for evergreen to download and include.  Not sure the best way to reword it but here's a suggestion:

To specify which version of [`libmongosqltranslate`](https://github.com/10gen/mongosql-rs) that should be built with the release, include the version of libmongosqltranslate in the tag name. This is done by annotating the tag with `libv`.